src/model_inspector/input_tester.py

```diff
@@ -106,32 +109,51 @@ def try_audio_input(self):
         for input_size in input_sizes:
             self.test_input_size(input_size)
 
-    def test_input_size(self, input_size):
-        output_size = is_valid_input_shape(self.model, input_size)
-        if output_size is not None:
-            self.working_input_sizes.append(input_size)
-            self.working_output_sizes.append(output_size)
-        else:
-            self.not_working_input_sizes.append(input_size)
+    def test_input_size(self, input_size, add_batch_dimension=False):
+        input_array = torch.ones(
+            (input_size)
+        ).numpy()  # i get type issues when using np.ones()
+        input_tensor = {"input": input_array}
+        output = None
+        if add_batch_dimension:
+            input_tensor.unsqueeze(0)
```

Comment:
the input_tensor is a dictionary, I think you mean to apply this on the value for the "input" entry. This will panic as its written